extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -475,7 +475,8 @@ export class AmpA4A extends AMP.BaseElement {
             try {
               // Among other things, the signature might not be proper base64.
               return verifySignature(
-                  response.creative, response.signature, publicKeyInfos);
+                  new Uint8Array(response.creative),
```

Comment:
Go for the demo :-).

But please make sure we have it tracked somewhere (maybe a TODO?).